### ISSUE TITLE
Suggestion: Rename `status` values in transfer response

### ISSUE THREAD

**Issue by showered** [kin-labs/mogami#199]
Whichever name we choose in version 1.0.0, we are stuck with until at least 2.0.0. For some reason `"Succeed"` doesn't sit well with me. I would prefer `"Success"` or `"success"`.

Just a thought!

**Comment by RamirezAlex** [comment]
Thanks for the feedback @showered. We'll change it.

**Comment by beeman** [comment]
This will be addressed in the work I am doing in #179

**Comment by beeman** [comment]
`Succeed` is no longer used. The `AppTransactionStatus` is now defined like this:

```prisma
enum AppTransactionStatus {
  Confirming
  Failed
  Finalized
  Pending
}
```

The order is first `Pending`, then `Confirming` -> `Finalized` or `Failed`.
